app/screens/interactive_dialog/interactive_dialog.js

```diff
@@ -106,17 +110,27 @@ export default class InteractiveDialog extends PureComponent {
 
         this.submitted = true;
 
-        if (!data || !data.errors || Object.keys(data.errors).length === 0) {
-            this.handleHide();
-            return;
+        let hasErrors = false;
+
+        if (data) {
+            if (data.errors &&
+                Object.keys(data.errors).length >= 0 &&
+                checkIfErrorsMatchElements(data.errors, elements)
+            ) {
+                hasErrors = true;
+                this.setState({errors: data.errors});
+            }
+
+            if (data.error) {
+                hasErrors = true;
+                this.setState({error: data.error});
+                this.scrollView.current.scrollTo({x: 0, y: 0});
```

Comment:
Make sure the reference exists before scrolling to the position

```suggestion
                if (this.scrollView?.current) {
                    this.scrollView.current.scrollTo({x: 0, y: 0});
                }
```